src/Pages/Login.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { NavBar } from "../Components/Nav";
```

Comment:
for importing components use may use single index file to export all componnts from it.